### DISCUSSION TITLE
1.6 release requirements or roadmap?

### DISCUSSION THREAD

**Discussion by FPDK** [thirtybees/ThirtyBees#1832] [General]
Hi,

I'm planning on updating a fairly old TB shop, but I'm wondering if I should wait for 1.6.

Is there a set list of changes/issues required before releasing a stable 1.6, or is it mainly resolving the current active issues?

I have seen the [README roadmap](https://github.com/thirtybees/thirtybees/commit/d9c85fa9ec25a186494daba2f3eec983ae75c553), but it has not been updated since October 2021.

**Comment by getdatakick** [comment]
I don't think you need to wait for 1.6 -- if it's an old thirty bees store, it's probably better to do incremental updates, and test in between.
